_data/projects/azure-sdk-for-js.yml

```diff
@@ -0,0 +1,13 @@
+name: Microsoft Azure SDK for JavaScript
+desc: This repository contains official JavaScript libraries and TypeScript definitions for Azure services.
+site: https://github.com/Azure/azure-sdk-for-js
+tags:
+- javascript
+- typescript
+- microsoft
+- azure
+- cloud
+- sdk
+upforgrabs:
+  name: Up for grabs
+  link: https://github.com/Azure/azure-sdk-for-js/labels/Up%20for%20grabs
```

Comment:
@srnagar There aren't any issues labeled with this tag.

Reply:
@ritwik12 This label was created yesterday to register with up-for-grabs. We will be tagging existing issues/creating new ones with this label.

Is it required that these labels must have issues before merging or can it be merged now and we will start tagging existing issues with this label soon?

Reply:
@srnagar  I am merging but make sure to add issues with this label as without it contributors will not be able to contribute.